### ISSUE TITLE
Listening Responses

### ISSUE THREAD

**Issue by SonicKim** [lukeburns/costa2018.com#21]
Hey Luke,

We're trying figure out where the Listening First responses are going and where we can see them.

When we open the google sheets, we see a name and time stamp but no response... Can you look into this?

Thank you!

**Comment by wamoyo** [comment]
Yeup, same on this end... 

![screenshot from 2017-09-14 10 50 10](https://user-images.githubusercontent.com/1170326/30436713-df3c82e0-993a-11e7-9a80-da1126053c1f.png)

**Comment by wamoyo** [comment]
I think the best thing we can do, is use the Github issues API to submit issues with labels from the website right to this repo. (then again, this might only work if people have github accounts. We might need to create an account called "Constituents")

**Comment by lukeburns** [comment]
Seems to be working over here. I see Costa's response. May just need to prevent form from submitting if is empty.

**Comment by wamoyo** [comment]
No, Sonic put in a full response, and it didn't go through.

**Comment by wamoyo** [comment]
I think my coach Bill might have too, that's his email in there.

**Comment by lukeburns** [comment]
It's def working for me. Tried on computer and phone.

**Comment by lukeburns** [comment]
If it's not working only sometimes...that worries me more.

**Comment by SonicKim** [comment]
It's only working sporadically. I just sumitted another one that didnt go through.

**Comment by wamoyo** [comment]
This might work: https://developers.google.com/apps-script/reference/forms/

**Comment by lukeburns** [comment]
Hmm thinking that only let's us create forms programmatically -- not sure it helps with how we present it :\

**Comment by wamoyo** [comment]
@lukeburns Seems the embed is still there. Any chance we can switch to link? Keeps the design consistent at least.

**Comment by lukeburns** [comment]
Kept playing with different link options, but gut tells me it'll reduce submissions. I changed design a bit to make it feel more cohesive.

**Comment by SonicKim** [comment]
Agreed. I think this works for now. Thanks for the thoughtful solve, Dr. Luke!
